packages/mobile/src/components/track-list/TrackListItem.tsx

```diff
@@ -89,7 +89,6 @@ export type TrackListItemProps = {
   hideArt?: boolean
```

Comment:
What are the changes in this file for?

Reply:
ah, yeah so we are not handling `isLoading` anymore, chatting it over with ray. it wasn't implemented, and since loading state is so short, were just not gonna deal with it